### ISSUE TITLE
Null values for unset/None properties in CouchDB

### ISSUE THREAD

**Issue by janua** [benoitc/couchdbkit#47]
Say we have the object Person, with two properties.

```
class Person(Document):
    name = StringProperty()
    age = IntegerProperty()

Person.set_db(db)
person = Person()
```

So we now have a blank person object.
When saving this;

```
person.save()
```

..the entry in CouchDB has a key/value pair for both 'name' and 'age' set to 'null'.
Now if I delete a property, then save it like this;

```
del person.name
person.save()
```

...the entry in CouchDB instead does not have the key/value pair for name. (Still has the age one).
Now when I check the values in person, I get this:

```
print person.name
None
print person.age
None
```

So basically, both are None, but still age shows up in CouchDB as null.
Is this normal behaviour?
Thanks!
